src/main/java/org/osiam/storage/query/GroupSimpleFilterChain.java

```diff
@@ -33,25 +31,25 @@
 
 public class GroupSimpleFilterChain implements FilterChain<GroupEntity> {
 
-    private final ScimExpression scimExpression;
+    private final FilterExpression filterExpression;
 
     private final GroupQueryField filterField;
     private final CriteriaBuilder criteriaBuilder;
 
-    public GroupSimpleFilterChain(CriteriaBuilder criteriaBuilder, ScimExpression scimExpression) {
+    public GroupSimpleFilterChain(CriteriaBuilder criteriaBuilder, FilterExpression filterExpression) {
         this.criteriaBuilder = criteriaBuilder;
-        this.scimExpression = scimExpression;
-        filterField = GroupQueryField.fromString(scimExpression.getField().toLowerCase(Locale.ENGLISH));
+        this.filterExpression = filterExpression;
+        filterField = GroupQueryField.fromString(filterExpression.getField().getFieldName());
```

Comment:
The filter field can be directly retrieved: `filterExpression.getField().getQueryField()`.